server/routes/shop.js

```diff
@@ -141,7 +129,31 @@ router.post(
       .send({ item: newItem });
   }
 );
-
+router.put("/details-update/:userid", verify, async (req, res) => {
```

Comment:
I feel like there is a better naming convention that is more RESTful for this, something like:
PUT `shops/:shop_id/details` or userid instead of shopid